easybuild/easyconfigs/h/Hadoop/Hadoop-2.6.0-cdh5.12.0-native.eb

```diff
@@ -0,0 +1,31 @@
+name = 'Hadoop'
+version = '2.6.0-cdh5.12.0'
+versionsuffix = '-native'
+
+homepage = 'http://archive.cloudera.com/cdh5/cdh/5/'
+description = """Hadoop MapReduce by Cloudera"""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+sources = ['%(namelower)s-%(version)s-src.tar.gz']
+source_urls = ['http://archive.cloudera.com/cdh5/cdh/5/']
+checksums = ['6eb2ff649ae1c5d1271732ca766b4e03728190ac4274ab00748ef8c44aa6e8d5']
+
+patches = ['Hadoop-TeraSort-on-local-filesystem.patch']
+
+builddependencies = [
+    ('Maven', '3.5.0'),
+    ('protobuf', '2.5.0'),  # *must* be this version
+    ('CMake', '3.9.1'),
+    ('snappy', '1.1.6'),
+]
+
+dependencies = [('Java', '1.7.0_80')]
```

Comment:
As I understood, yes, Java 8 is only supported in Hadoop 2.7 & later, I think.

I seem to remember trying with `1.8.0_*`, and running into compilation problems...